be/src/exec/aggregator.cpp

```diff
@@ -34,7 +34,34 @@ namespace starrocks {
 Status init_udaf_context(int64_t fid, const std::string& url, const std::string& checksum, const std::string& symbol,
                          FunctionContext* context);
 
-Aggregator::Aggregator(const TPlanNode& tnode) : _tnode(tnode) {}
+std::shared_ptr<AggregatorParams> convert_to_aggregator_params(const TPlanNode& tnode) {
+    auto params = std::make_shared<AggregatorParams>();
+    params->conjuncts = tnode.conjuncts;
+    params->limit = tnode.limit;
+
+    switch (tnode.node_type) {
+    case TPlanNodeType::AGGREGATION_NODE: {
```

Comment:
node_type is TPlanNodeType::AGGREGATION_NODE, it is always a truth,   if not so, it is  a fatal error, so use DCHECK instead.

Reply:
`STREAM_AGGREGATION_NODE` will be added later for Incremental MV, so here add a PlanNode to AggregatorParams function .